packages/@uppy/companion/src/server/Uploader.js

```diff
@@ -385,7 +389,15 @@ class Uploader {
     const formData = Object.assign(
       {},
       this.options.metadata,
-      { [this.options.fieldname]: file }
+      {
+        [this.options.fieldname]: {
+          value: file,
+          options: {
+            filename: this.uploadFileName,
+            contentType: this.options.metadata.type
+          }
+        }
+      }
```

Comment:
Maybe a comment/link here too, as to why `filename` and `contentType`? Stating that it’s conventional and linking to https://github.com/form-data/form-data? To be honest, I didn’t find docs that explicitly stated that these fields should be used, only examples.

Reply:
I think it might be redundant because `Content-Disposition` is pretty common and that's what the browser also uses, and the library used is `request` so linking to `request` is unnecessary because this exact example is in the readme.

Reply:
Ok, I’ll leave a link here then: https://github.com/request/request#multipartform-data-multipart-form-uploads